zipline/lib/adjusted_array.py

```diff
@@ -165,6 +166,46 @@ def __init__(self, data, adjustments, missing_value):
         self.adjustments = adjustments
         self.missing_value = missing_value
 
+    def append_adjustments(self, adjustments_to_append, append_back):
```

Comment:
Two minor naming notes:

1. I would rename `adjustments_to_append` to just `adjustments`. It's clear from context what's happening to them.
2. The name `append` here is a bit confusing. "append" usually means "add to the end", so having a flag to say "append to front" or "append to back" sounds off to me. Something like `add`/`merge`/`update_adjustments` would make more sense to me here.